test/runtests.jl

```diff
@@ -5,36 +5,46 @@ using DataFrames
 using Random
 using MLJTestInterface
 using Statistics
+using Clustering
 using Distances
-using StatsBase: countmap
+using StatsBase: countmap, mode
 using NearestNeighbors, Distances
 using MLJBase: machine, transform
 using StableRNGs: StableRNG
 using TableTransforms
 using TransformsBase
 using ScientificTypes
 using IOCapture
-ENV["PYTHON"]=""
-import Pkg; Pkg.build("PyCall")
-using PyCall       
-using Conda                 
-
-
+ENV["PYTHON"] = ""
+import Pkg;
+Pkg.build("PyCall");
+using PyCall
+using Conda
 
 include("test_utils.jl")
 @testset "common_utils" begin
     include("common_utils.jl")
 end
 
+@testset "extras" begin
+    include("extras.jl")
+end
+
 @testset "table_wrappers" begin
     include("table_wrappers.jl")
 end
+
+@testset "generic_resample" begin
+    include("generic_resample.jl")
+end
+
 @testset "tabletransforms" begin
     include("tabletr_interface.jl")
 end
 @testset "MLJ Interface" begin
     include("mlj_interface.jl")
 end
+#=
```

Comment:
This looks like a bunch of testing has been disabled. Is this intentional?

Reply:
Oops. It definitely is not, I just tend test only the files I changed before a commit to avoid waiting for too long. Ideally, I uncomment them back and let the Github action do it all again which logically succeeds because I tested the changed files.